Xamarin.PropertyEditing.Mac/Controls/Custom/NumericTextField.cs

```diff
@@ -156,14 +172,16 @@ public override void DidEndEditing (NSNotification notification)
 		}
 	}
 
-	internal class KeyUpDownDelegate : NSTextFieldDelegate
+	internal class KeyUpDownDelegate : TextNextResponderDelegate
 	{
 		public event EventHandler<bool> KeyArrowUp;
 		public event EventHandler<bool> KeyArrowDown;
 
 		public override bool DoCommandBySelector (NSControl control, NSTextView textView, Selector commandSelector)
 		{
-			switch (commandSelector.Name) {
+			if (!base.DoCommandBySelector(control, textView, commandSelector))
```

Comment:
For a second there, I thought this called `NSTextFieldDelegate.DoCommandBySelector`, which would've thrown an exception.

Reply:
I added a comment explaining better how the base handles the commandselector

Reply:
fixed